### PR TITLE
[NO SQUASH] Remove ancient debug comments

### PR DESCRIPTION
**Pull request by otakubeam**
I hope it's nothing too important

### CODE REVIEW

**Comment by otakubeam** [comment]
I just realized applying clang-format is unnecessary because it would just muddle git blame output

**Comment by sfan5** [comment]
Indeed, please don't apply clang-format to files you changed.
Also feel free to grep the entire codebase for any commented out stuff that prints to `dstream`.

**Comment by SmallJoker** [comment]
Fine by me, as long the PR is not squashed.

**Comment by appgurueu** [comment]
@Zughy I don't see which action would be needed? The commits are neatly separated, have decent commit messages (IMO) and  were apparently considered fine by @SmallJoker.

**Comment by Zughy** [comment]
@appgurueu My interpretation of `but requires good commit messages` is that right now those names aren't good. Was I wrong? :thinking:

File: src/voxel.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "nodedef.h"
 #include "util/directiontables.h"
 #include "util/timetaker.h"
-#include <cstring>  // memcpy, memset
+#include <cstring> // memcpy, memset
```
Comment (nerzhul) [inline src/voxel.cpp:26]: you can also drop those comments, IDE are here for that, no need to comment headers

File: src/voxel.cpp [hunk 2]
```diff
@@ -278,43 +246,12 @@ void VoxelManipulator::clearFlag(u8 flags)
...
-	}
-
-	dstream<<"clearFlag changed "<<count<<" flags out of "
-			<<volume<<" nodes"<<std::endl;*/
 }
 
 const MapNode VoxelManipulator::ContentIgnoreNode = MapNode(CONTENT_IGNORE);
 
-//END
+// END
```
Comment (nerzhul) [inline src/voxel.cpp:257]: i don't see the point to keep this comment too, a comment to define end of the file ? :p

**Review by nerzhul — CHANGES_REQUESTED** [review]

**Comment by Zughy** [comment]
OP has disappeared, closing

**Pull request closed without merging**